wire-ios/Wire-iOS Tests/CreateSecureGuestLinkViewModelTests.swift

```diff
@@ -17,34 +17,82 @@
 //
 
 import XCTest
+import WireCommonComponents
 @testable import Wire
 
-class CreatePasswordSecuredLinkViewModelTests: XCTestCase {
+class CreateSecureGuestLinkViewModelTests: XCTestCase {
 
     // MARK: - Properties
 
     var viewModel: CreateSecureGuestLinkViewModel!
     var mockDelegate: MockCreatePasswordSecuredLinkViewModelDelegate!
+    var textField: ValidatedTextField!
+    var confirmPasswordField: ValidatedTextField!
 
     // MARK: - setUp
 
     override func setUp() {
         super.setUp()
+        FontScheme.configure(with: .large)
```

Comment:
As I've suggested in the past, this setup could be done in a base test class which all tests inherit.

Reply:
Done!